public/assets/tokens/evmos.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="940" height="660" viewBox="0 0 940 660" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please sync default appearance of SVG to match PNG

Reply:
synced